### PR TITLE
Order admin

### PR DESCRIPTION
**Pull request by namuetn**
## Related Tickets:
- [#14 28934](https://edu-redmine.sun-asterisk.vn/issues/28934)

## WHAT this PR does?
- display and change status order page
## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by namuetn** [comment]
ready

File: resources/views/fashi/admin/order/index.blade.php [hunk 1]
```diff
@@ -0,0 +1,136 @@
+@extends('admin_master')
+
+@section('content')
+<br>
```
Comment (thaonp104) [inline resources/views/fashi/admin/order/index.blade.php:4]: 4 spaces

File: resources/views/fashi/admin/order/index.blade.php [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+                                            <thead>
+                                                <tr>
+                                                    <th scope="col">#</th>
+                                                    <th scope="col">{{ trans('text.details') }}</th>
+                                                    <th scope="col">{{ trans('text.price') }}</th>
+                                                </tr>
+                                            </thead>
+                                            <tbody>
+                                                @foreach ($order->orderDetails as $orderDetail)
+                                                <tr>
```
Comment (thaonp104) [inline resources/views/fashi/admin/order/index.blade.php]: 4 spaces

File: resources/views/admin_master.blade.php [hunk 3]
```diff
@@ -14,6 +14,7 @@
     </head>
 
     <body class="hold-transition sidebar-mini">
+            @include('sweetalert::alert')
```
Comment (ghost) [inline resources/views/admin_master.blade.php]: indent 4 spaces

File: resources/views/fashi/admin/order/index.blade.php [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+                                                    </tr>
+                                                @endforeach
+                                            </tbody>
+                                        </table>
+                                    </div>
+
+                                    <div class="modal-footer">
+                                        <form method="POST" action="{{ route('admin.orders.pending', $order->id) }}">
+                                            @csrf
+                                            <button type="submit" @if ($order->status == config('order.pending')) disabled="" @endif class="btn btn-info">{{ trans('text.pending') }}</button>
```
Comment (ghost) [inline resources/views/fashi/admin/order/index.blade.php]:
```suggestion
                                            <button type="submit" @if ($order->status === config('order.pending')) disabled="" @endif class="btn btn-info">{{ trans('text.pending') }}</button>
```

File: resources/views/fashi/admin/order/index.blade.php [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+                                        </form>
+
+                                        <form method="POST" action="{{ route('admin.orders.success', $order->id) }}">
+                                            @csrf
+                                            <button type="submit" @if ($order->status == config('order.success')) disabled="" @endif class="btn btn-success">{{ trans('text.success') }}</button>
```
Comment (ghost) [inline resources/views/fashi/admin/order/index.blade.php]: ở dưới cũng để ===  hết đi ô

**Comment by namuetn** [comment]
ready

**Pull request merged**